homeassistant/components/keyboard_remote/manifest.json

```diff
@@ -3,6 +3,7 @@
   "name": "Keyboard Remote",
   "codeowners": ["@bendavid", "@lanrat"],
   "documentation": "https://www.home-assistant.io/integrations/keyboard_remote",
+  "integration_type": "hub",
```

Comment:
Please make a separate PR for this addition.

Reply:
done.